spring-cloud-dataflow-core/src/test/java/org/springframework/cloud/dataflow/core/dsl/StreamParserTests.java

```diff
@@ -451,10 +432,10 @@ public void errorCases06() {
 
 	@Test
 	public void errorCases07() {
-		checkForParseError("foo > bar", DSLMessage.EXPECTED_CHANNEL_PREFIX_QUEUE_TOPIC, 6, "bar");
-		checkForParseError("foo >", DSLMessage.OOD, 5);
-		checkForParseError("foo > --2323", DSLMessage.EXPECTED_CHANNEL_PREFIX_QUEUE_TOPIC, 6, "--");
-		checkForParseError("foo > *", DSLMessage.UNEXPECTED_DATA, 6, "*");
+		//checkForParseError("foo > bar", DSLMessage.EXPECTED_CHANNEL_PREFIX, 6, "bar");
```

Comment:
yeah, in this case it is tricky to suggest the exact DSL error message. For this definition `foo > bar`, foo is parsed as Module Node and since there is GT token the `bar` is expected to be a named destination. 
1) We could either say, "Expected Channel Prefix" at the `bar` position 
2) but `foo` could also be a named destination. 

May be it is ok to say (1) here? If so, [this] (https://gist.github.com/ilayaperumalg/284d39af2a31dfc8a5db) can be the change we need.